assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,24 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId) {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
I'd normally make something more general here, e.g:
var focusElement = document.querySelector(this.getAttribute('data-focuses'));

which would mean that you could use any css selector to target an element rather than just id.